__test__/integration/loan_offer/runners/accept.ts

```diff
@@ -0,0 +1,95 @@
+import * as Web3 from "web3";
+
+import { BigNumber, Dharma } from "../../../../src";
+
+import { LoanOffer } from "../../../../src/types";
+
+import { ContractsAPI } from "../../../../src/apis/";
+
+import { DummyTokenContract } from "../../../../src/wrappers";
+
+// Accounts
+import { ACCOUNTS } from "../../../accounts";
+
+const { Token } = Dharma.Types;
+
+const CREDITOR = ACCOUNTS[0];
+const DEBTOR = ACCOUNTS[1];
+
+const TX_DEFAULTS = { from: CREDITOR.address, gas: 4712388 };
+
+export async function testAccept(
+    dharma: Dharma,
+    params: any,
+    signAndAccept: (loanOffer: LoanOffer, address: string) => void,
```

Comment:
address here is vague. who is this? debtor? creditor?

Reply:
Done.